wpilibNewCommands/src/main/native/cpp/frc2/command/button/CommandGenericHID.cpp

```diff
@@ -75,6 +75,13 @@ Trigger CommandGenericHID::AxisGreaterThan(int axis, double threshold,
   });
 }
 
+Trigger CommandGenericHID::AxisMagnitudeGreaterThan(
+    int axis, double deadband, frc::EventLoop* loop) const {
+  return Trigger(loop, [this, axis, deadband]() {
+    return std::abs(m_hid.GetRawAxis(axis)) > deadband;
+  });
+}
```

Comment:
`deadband` should be renamed to `threshold` to match Java and the declaration in the header file.